packages/lib/gpcircuits/circuits.json

```diff
@@ -12,12 +12,11 @@
       0
     ],
     "pubs": [
-      "objectSignerPubkeyAx",
```

Comment:
When I realized I hadn't seen any update to the code to stop always revealing the signer's public key, it took me a while to realize this was where that change is expressed.
I dislike how circom makes the distinction between public and private inputs so obscure and separate from the code itself.  It's also very asymmetric from the distinction of outputs (public derived signals) from private derived signals, which is specified in the code.
(No action here, just commenting.)